### PR TITLE
Add bootstrap 4 + reactstrap primitives

### PR DESCRIPTION
**Pull request by craig-mulligan**
Connects to #13

This is pretty massive (sorry). Have all basic bootstrap functionality, but now I'm trying to map our style guide to bootstrap variables. 

It may make sense to change the style guide slightly (interms of naming) to suite our needs. eg. Primary color in bootstrap is used for default link color so we should probably swap that with the secondary. There are some other considerations too, worth jumping on a call for that if you have opinions too. 

Change-Type: patch

### CODE REVIEW

**Comment by craig-mulligan** [comment]
ping @LucianBuzzo. 

I'm waiting for the exact measurements to do the styling etc. But this adds a good base we can perfect the styling later.

**Comment by craig-mulligan** [comment]
PingyPong @LucianBuzzo when you have a sec.

File: src/styles/variables.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+// BOOTSTRAP SEMANTICS
+$brand-primary: $blue;
+
+// CUSTOM SEMANTICS
+$brand-secondary: $yellow;
+$brand-tertiary: $gray-dark;
+
+// FONTS
+$font-family-sans-serif: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial;
```
Comment (LucianBuzzo) [inline src/styles/variables.scss]: After recent discussion, we're going to be using Roboto (same as the dashboard) so this should probably be changed.

**Comment by craig-mulligan** [comment]
Updated @LucianBuzzo

**Pull request merged**